app/controllers/pipeline_work_in_progress_controller.rb

```diff
@@ -0,0 +1,144 @@
+# frozen_string_literal: true
+
+class PipelineWorkInProgressController < ApplicationController
+  def index
+    # TODO: how can we avoid hardcoding this? Problem is there are multiple relevant pipelines we want to display in one.
+    @pipeline = '"Heron"'
+
+    # TODO: test including the 'Heron 384 Tailed MX' pipeline - might cause an issue as there might be loads of tubes in the final purpose
+    pipeline_configs = Settings.pipelines.select{ |pipeline| ['Heron-384 Tailed A', 'Heron-384 Tailed B'].include? pipeline.name }
+    @ordered_purpose_list = combine_and_order_pipelines(pipeline_configs)
+
+    page_size = 500
+    param_date = params[:date]&.to_date
```

Comment:
I think you can just do
`from_date = params[:date]&.to_date || Date.today.prev_month`

Reply:
yes that's neater, thanks